src/ripple/server/ServerHandler.h

```diff
@@ -96,7 +96,10 @@ class ServerHandler
 //------------------------------------------------------------------------------
 
 ServerHandler::Setup
-setup_ServerHandler (BasicConfig const& c, std::ostream& log);
+setup_ServerHandler (
+    BasicConfig const& c,
+    bool acceptPeers,
```

Comment:
I'd love to get rid of this `acceptPeers` param. I don't have a better solution, but we should try to improve this.